backend/dataall/modules/redshift_datasets/services/redshift_connection_service.py

```diff
@@ -46,10 +52,14 @@ def create_redshift_connection(uri, admin_group, data: dict) -> RedshiftConnecti
                 database=data.get('database'),
                 redshiftUser=data.get('redshiftUser', ''),
                 secretArn=data.get('secretArn', ''),
+                connectionType=data.get('connectionType', RedshiftConnectionTypes.DATA_USER.value),
```

Comment:
does not look like we pass a parameter for `connectionType` - so will every connectionType be defaulted to `RedshiftConnectionTypes.DATA_USER.value`?